packages/theme/src/theme/_buttons.scss

```diff
@@ -78,13 +78,16 @@
 		&.btn-icon-only {
 			color: $gray;
 
-			&:focus,
 			&:hover {
-				color: $scooter;
+				svg {
+					fill: $scooter;
+				}
 			}
 
 			&:active {
-				color: darken($scooter, 12%);
+				svg {
+					fill: darken($scooter, 12%);
+				}
```

Comment:
What about this ?
```css
			/* Existing code */
			&:focus,
			&:hover {
				color: $scooter;
			}

			&:active {
				color: darken($scooter, 12%);
			}
			/* End of existing code */

			svg {
				fill: currentColor;
			}
```

Reply:
Following the guideline, icons should not have `currentColor`, but `$silver-chalice`.
See http://guidelines.talend.com/document/92132#/ui-controls/buttons/icon-buttons

Reply:
We have (in a line just above my copy/paste) `.btn-icon-only { color: $gray; }`. Is this correct ?
We should either
* remove all color property and style the svg in `.btn-icon-only`
or
* manage the color property and set svg fill to currentColor

Reply:
Finally removed because globally defined here
https://github.com/Talend/ui/blob/e73302d4644d077aff4d3833adca36730f89e82a/packages/theme/src/theme/_buttons.scss#L32

Reply:
<img width="496" alt="Capture d’écran 2019-04-02 à 16 56 10" src="https://user-images.githubusercontent.com/10761073/55412729-44c17680-5568-11e9-9d18-de07c38e1ff1.png">
That made a regression.

Let's step back a little.
* you have a svg fill set to currentColor (once you revert the line). It makes sense because the icon should fit the text color
* **on btn-link and icon-only combined**, you have a specific case, where it must be `$silver-chalice`. You have 2 choices: set color to `$silver-chalice`, so the svg is filled with (currentColor), or set the svg fill. You have 2 other cases (hover and active) in this case. Stay with the same way to do it (color of svg fill).

Reply:
Another point, you removed the style on focus. Is it what is wanted ?
For a11y, you still have the outline that shows that you have focus on it, so it's acceptable, but need to be sure that it's what UX wants.

Reply:
Too much dependencies :dizzy_face:
Let's fix it !

It looks strange to me to have blue icon if active, but I will ask Ulcay about that 👍